proxy/api/protocol.go

```diff
@@ -28,6 +29,18 @@ type header struct {
 	flags  uint32
 }
 
+const maxPayloadLength = 1 * 1024 * 1024
```

Comment:
- I guess it does add a little clarity even though redundant (currently).
- Roger that.